include/shell/shell.h

```diff
@@ -71,6 +71,22 @@ struct shell_cmd_entry {
 
 struct shell;
 
+/**
+ * @brief Initializes a shell command arguments
+ *
+ * @param[in] _mandatory Number of mandatory arguments.
+ * @param[in] _optional  Number of optional arguments.
+ */
+#define SHELL_ARG(_mandatory, _optional) {				\
+	.mandatory = _mandatory,					\
+	.optional = _optional,						\
+}
+
+struct shell_static_args {
```

Comment:
Argument `argc` in command handler is `size_t` type, just a proposal to change these parameters to `size_t` as well.

Reply:
Initially I was using that but it takes a lot of space, besides size_t + size_t may overflow (not that I think someone will be crazy enough to have this manu arguments).